siliconcompiler/apps/sc_configure.py

```diff
@@ -70,7 +70,7 @@ def main():
 
     server = urlparse(srv_addr)
     has_scheme = True
-    if not server.scheme:
+    if not server.hostname:
```

Comment:
Won't it just return early 5 lines down, when `if not server.hostname:` is checked again?

Reply:
The second check is the last check. Then it's first checked it's to see if the url was `localhost` or `https://localhost`, if it was the latter, the parsing worked correctly, if not, then we need to make it appear as if it is a real URL. If the parsing still fails to find the hostname then we exit out.

Reply:
Oh, I see - `server` is a urlparse object which has a `hostname` property that gets updated when `urlparse` is called.